### PR TITLE
Fix #109 (position of cardboard UI when predistorted:true)

### LINKED ISSUE #109: predistorted: true makes the cardboard UI too big and wrongly positioned
When you use `predistorted: true` in the example page (by modifying line 176 of the `index.html`):
```
vrDisplay.requestPresent([{source: renderer.domElement, predistorted: true}]);
```
You end up with a weird cardboard UI.

This is when you click on the cardboard icon from portrait mode:
screenshot_20160512-200722

This is when you click on the cardboard icon from landscape mode.
screenshot_20160512-200737

### PR DESCRIPTION
**Pull request by idlewan**

### CODE REVIEW

**Comment by borismus** [comment]
Looks good, thanks.

**Comment by borismus** [comment]
Oops somehow closed by accident. Looks like this has some conflicts. Could you merge to master and resubmit?

**Comment by idlewan** [comment]
Oh, sorry, I was based on master from a few days ago. Just rebased with the current master, should be better now.

**Comment by borismus** [comment]
Have you tested this? Doesn't work for me.

File: build/webvr-polyfill.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ var currentQueue;
 var queueIndex = -1;
 
 function cleanUpNextTick() {
+    if (!draining || !currentQueue) {
```
Comment (cvan) [inline build/webvr-polyfill.js]: this seems to be have been added manually?

Comment (borismus) [inline build/webvr-polyfill.js]: Nah, that's some artifact of browserify

**Comment by idlewan** [comment]
I've tested it with the example `index.html`. It is positionned correctly and scaled correctly when clicking the VR icon from landscape (I'm testing this on the Oneplus One and Samsung S6). From portrait orientation, it seems positionned correctly but the scale of the back and gear icon and the line is too big.

At the time I did that pull request, I didn't find the proper way to fix this when starting from portrait mode, but it still was an improvement so I just submitted the pull request.

**Comment by borismus** [comment]
It wasn't working with me, had to introduce a different workaround.

**Pull request merged**